examples/with-portals-ssr/pages/index.js

```diff
@@ -1,4 +1,3 @@
-import * as React from 'react'
 import { UniversalPortal } from '@jesstelford/react-portal-universal'
 
 export default class Index extends React.Component {
```

Comment:
This code is no longer valid. Can you re-add importing `Component`?

Reply:
At first sight, it seems like it should not work, but I test basically every change I make and the current example builds and runs in dev and production, so I believe the react-portal-universal package is importing React too or something else I cant catch.

If you insist, I can still import the Component, but I just wanted to share what I have investigated before doing so.

Reply:
@Timer, just confirmed again that it builds, serves dev and production.

Reply:
This won't pass standard linting so we should import it:
```
import { Component } from 'react'
```